### PR TITLE
Implements fine-grained data directories

### PR DESCRIPTION
**Pull request by lwrubel**
Based on @SvenLieber's work in #1055 to use data directories on mounted filesystems. See https://github.com/gwu-libraries/sfm-docker/pull/30 for PRs for other repos which must be checked out as well to test.

### CODE REVIEW

**Comment by lwrubel** [comment]
There's a problem with how warc paths are handled. For WARCs created before the fine-grained directories change, the path to each warc is stored in the database as `/sfm-data/collection_set/<collection set id>/<collection id>/` etc.

After the update, the path stored changes to:
`/sfm-collection-set-data/collection_set/<collection set id>/<collection id>/` etc.

This causes errors when exporting because the exporter is reading the paths from the database and not finding `/sfm-data/collection_set`.

**Comment by lwrubel** [comment]
Addressed issue with legacy internal paths by adding to .env `DATA_VOLUME_FORMER_COLLECTION_SET` and
`DATA_VOLUME_FORMER_EXPORT`

File: docs/storage.rst [hunk 1]
```diff
@@ -6,10 +6,13 @@
  Storage volumes
 -----------------
 
-SFM stores data on 2 volumes:
+SFM stores data on multiple volumes:
```
Comment (kerchner) [inline docs/storage.rst:9]: Perhaps "in multiple directories, which may be set up as separate volumes"

**Review by kerchner — COMMENTED** [review]
Looks good - one very mild suggestion, which you might opt to pass on

**Pull request merged**